training/automl/components/automl_image_classification/spec.yaml

```diff
@@ -1,7 +1,7 @@
 $schema: http://azureml/sdk-2-0/AutoMLComponent.json
 name: microsoft_azureml_automl_image_classification_component
-display_name: AutoML Image Classification (Private Preview)
+display_name: AutoML Image Classification
 description: Component that executes an AutoML Image Classification task model training in a pipeline.
-version: 0.0.1-preview
+version: 0.0.1
```

Comment:
We should verify that this is the versioning process used by other Designer components.

Reply:
For components, we have in azureml-assets, this is being followed except in case of benchmarking where they have versioning such as `1`, `1.0.0`. Can you please help to confirm which one we should go with?

Reply:
Discussed offline. updated version to 1.0.0